src/App.js

```diff
@@ -5,9 +5,11 @@ import {
   NavMenu,
   Story,
   StoryImage,
+  // Wedding,
```

Comment:
Might be worth removing these comments. They should be tickets if there's additional work to do here in the future

Reply:
Will do! Done.